vars/utils.groovy

```diff
@@ -21,6 +21,19 @@ def getCosaDir(params = [:]) {
     }
 }
 
+// Clone a git repository in the following dir:
+//    dir:               string -- Directory
+//    git_ref:           string -- Git branch
+//    git_uri:           string -- Git URL
+def gitClone(params = [:]) {
+    def path = params.get('dir','');
+    def branch = params.get('git_uri','');
+    def repo = params.get('git_ref', '');
+    dir(path) {
```

Comment:
Does this work if `path` is the default empty string?

I'm unsure also whether this thin wrapper is worth it. It doesn't seem to provide much value. Given that we usually only clone once per job, just letting the caller use the `git` step or call out to `git clone` seems OK. But fine too if you'd really like to have it!